general/networking/caddy.md

```diff
@@ -60,7 +60,8 @@ reverse_proxy 127.0.0.1:8096
 You can serve Jellyfin only at a particular base path and not proxy all other requests.
 
 To do this, first configure Jellyfin to use a base path.
-Go to `Admin > Networking` and in the Base URL field, enter a path like `/jellyfin`.
+If you already have access to the web interface, go to `Admin > Networking` and in the Base URL field, enter a path like `/jellyfin`.
+If not, you may instead go to `<Configuration_directory>/network.xml` and modify key `<BaseUrl>` to your needs. For references on the directory location, please consult [Configuration Documentation](https://jellyfin.org/docs/general/administration/configuration.html#configuration-directory).
```

Comment:
```suggestion
If not, you may instead go to `<configuration_directory>/network.xml` and modify the value of `<BaseUrl>` according to your needs. For information on the directory location, please consult the [configuration documentation](https://jellyfin.org/docs/general/administration/configuration.html#configuration-directory).
```

Reply:
Thanks for the kind suggestions, I agree with all of the proposed adjustments except one.

As I read through the documentation again, I realized the way of expressing`<configuration_directory>` that is the most consistent with the current [configuration documentation](https://jellyfin.org/docs/general/administration/configuration.html) is `<Configuration Directory>`.

Reply:
Good point, agreed.